### PR TITLE
Backporting to 71X: adding Pythia8 Filter for genparticles HT

### PR DESCRIPTION
**Pull request by alefisico**
I want to backport this request #12516 to 71X.
Please let me know if you have any question.

### CODE REVIEW

**Comment by covarell** [comment]
+1

**Comment by davidsheffield** [comment]
Can you merge this?

**Comment by covarell** [comment]
Hmmm... I believe this was already presented at the ORP on Jan 5? @davidlange6

**Comment by davidsheffield** [comment]
OK. I forget to check if it had been brought up in that meeting.

File: GeneratorInterface/GenFilters/src/PythiaFilterHT.cc [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+		cout << " MaxPhi = " << maxphicut << endl;*/
+		cout << " MinHT = " << minhtcut << endl;
+		cout << " MotherID = " << motherID << endl;
+
+}
+
+
+PythiaFilterHT::~PythiaFilterHT() {
+	std::cout << "Total number of tested events = " << theNumberOfTestedEvt << std::endl;
+	std::cout << "Total number of accepted events = " << theNumberOfSelected << std::endl;
```
Comment (davidlange6) [inline GeneratorInterface/GenFilters/src/PythiaFilterHT.cc]: @alefisico 
Sorry for the slow review

Please remove the couts. Thanks

**Comment by alefisico** [comment]
done, I remove the couts.

**Comment by alefisico** [comment]
I think we need them for validation, but if you think they are unnecessary and it will move this request forward, I've just removed them.

**Comment by alefisico** [comment]
Hi all, sorry to bother again, but is there a reason why this PR is not approved? Please let me know if I need to make any other change.

**Comment by covarell** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**